### PR TITLE
#167313378 Users should be able to report an article

### PR DESCRIPTION
**Pull request by minega25**
#### What does this PR do?
#167313378 Users should be able to report an article
#### Description of Task to be completed?
As an authenticated user I should be able to report articles that violate terms of the agreement or are plagiarised to the authors haven admin for removal and or any other actions as per the terms of engagement.
#### What are the relevant pivotal tracker stories?
[#167313378](https://www.pivotaltracker.com/story/show/167313378)
#### Screenshots (if appropriate)
Screen Shot 2019-11-08 at 11 46 01
Screen Shot 2019-11-08 at 11 46 09
Screen Shot 2019-11-08 at 11 46 15
Screen Shot 2019-11-11 at 12 51 00
Screen Shot 2019-11-11 at 12 51 05
Screen Shot 2019-11-11 at 12 51 08

### CODE REVIEW

File: src/components/Routes.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const Routes = () => (
     <Route exact path="/google/social-login" component={verifyAuth} />
     <Route exact path="/twitter/social-login" component={verifyAuth} />
     <Route exact path="/facebook/social-login" component={verifyAuth} />
-    <Route path={/\/reset/} component={ResetRequest} />
+    <Route path="/reset" component={ResetRequest} />
```
Comment (SokoPaulSokool) [inline src/components/Routes.js]: don't change this path

File: src/components/common/listOfArticles/index.js [hunk 2]
```diff
@@ -23,6 +23,12 @@ export default function ListOfArticles({ articles, limitArticlesNumber }) {
           slug={article && article.slug}
           rating={rating}
         />
+        {article.reportReason && (
+          <div className="alert alert-danger" role="alert">
+  Reported due to this reason:
+            {article.reportReason }
+          </div>
+        )}
```
Comment (SokoPaulSokool) [inline src/components/common/listOfArticles/index.js]: Put this inside article card at the bottom

**Review by SokoPaulSokool — CHANGES_REQUESTED** [review]

File: src/components/common/listOfArticles/index.js (hunk 2 above)
Comment (minega25) [reply src/components/common/listOfArticles/index.js]: Cool.

File: src/components/Routes.js (hunk 1 above)
Comment (minega25) [reply src/components/Routes.js]: This path was bringing invalid path warning in the console

File: src/components/common/listOfArticles/index.js (hunk 2 above)
Comment (minega25) [reply src/components/common/listOfArticles/index.js]: Worked on this

File: src/components/Routes.js (hunk 1 above)
Comment (minega25) [reply src/components/Routes.js]: Worked on this as well

**Pull request merged**